changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "3.0.0": [
+      "[New] Add support for notifications of pull request reviews - #14175",
```

I think this could use a little work and should read less like an objective description of what's added and more like a headline

```suggestion
      "[New] Get notified when your pull request is reviewed - #14175",
```

Could use more polish but that's closer to what I want

---

I like that! But I think it looks better using "pull requests" (in plural)
```suggestion
      "[New] Get notified when your pull requests are reviewed - #14175",
```